libs/api/domains/translations/src/lib/translations.service.ts

```diff
@@ -35,21 +41,24 @@ export class TranslationsService {
 
     const result = await this.contentfulRepository
       .getLocalizedEntries<any>('*', {
-        ['content_type']: 'namespace',
-        select: 'fields.strings,fields.fallback',
+        ['content_type']: 'namespaceJeremyDev', // TODO: replace after review
+        select: 'fields.strings',
         'fields.namespace[in]': namespaces.join(','),
       })
       .catch(errorHandler('getNamespace'))
 
-    const withFallbacks = result?.items?.map(({ fields }) =>
-      mergeWith(
-        {},
-        locale.fallbackCode ? fields.strings[locale.fallbackCode] : {},
-        fields.strings[locale.code],
-        (o, s) => (isEmpty(s) ? o : s),
-      ),
+    return (
+      result?.items?.map(({ fields }) =>
+        fields.strings['is-IS'].reduce(
+          (acc: Record<string, string>, cur: DictArray) => ({
+            ...acc,
+            [cur.id]: isEmpty(cur[locale.code])
+              ? cur?.[locale.fallbackCode]
+              : cur?.[locale.code],
+          }),
+          {},
+        ),
+      )?.[0] ?? null
```

Comment:
Is this only returning the first namespace requested?

Reply:
I realized that a template can use more than one namespace, I need to change that yep 👍